lib/llvm-backend/build.rs

```diff
@@ -262,6 +262,46 @@ fn is_llvm_debug() -> bool {
     llvm_config("--build-mode").contains("Debug")
 }
 
+fn get_llvm_target_name() -> String {
+    let name = if cfg!(all(target_os = "macos", target_arch = "x86_64")) {
+        "x86_64-apple-darwin"
+    } else if cfg!(all(target_os = "linux", target_arch = "x86_64")) {
+        "x86_64-linux-gnu-ubuntu-16.04"
+    } else {
+        panic!("Unsupport host for installing llvm")
+    };
+
+    format!("clang+llvm-8.0.0-{}", name)
```

Comment:
Please update to 8.0.1?

Reply:
I'm using pre-build binaries but it doesn't seem to exist 8.0.1 for Mac and Ubuntu 16.0.4 in the download page (http://releases.llvm.org/download.html).